devbot/tools/filewrapper.py

```diff
@@ -0,0 +1,18 @@
+"""
+    A wrapper for files so we can send files easily
+"""
+
+import os.path
+
+
+class FileWrapper:
```

Comment:
I suggest renaming this class to `File`. Applying the class in a command like this 
```
def command():
    return File("file/path")
```
 conveys its usage better.

Reply:
Maybe name the module `wrap.py`.
Then you can do:
```
import devbot.tools.wrap as wrap

file = wrap.File("file/path")
```

Added advantage, if we ever want to wrap something else in the future, we can add it to this module.